assets/src/components/SelectionTool.js

```diff
@@ -117,12 +210,38 @@ export default class SelectionTool extends HTMLElement {
 
         mainEventDispatcher.addListener(
             () => {
-                render(mainTemplate(), this);
+                render(
+                    mainTemplate(
+                        this.isFilterDisabled,
+                        this.results,
+                        mainLizmap.selectionTool.newAddRemoveSelected,
+                        mainLizmap.selectionTool.filteredFeaturesCount
+                    ),
+                    this
+                );
             },
-            ['selectionTool.newAddRemoveSelected', 'selectionTool.allFeatureTypeSelected', 'selection.changed', 'filteredFeatures.changed', 'selection.bufferValue']
+            [
+                'selectionTool.newAddRemoveSelected',
+                'selectionTool.allFeatureTypeSelected',
+                'selection.changed',
+                'filteredFeatures.changed',
+                'selection.bufferValue'
+            ]
         );
     }
 
     disconnectedCallback() {
     }
+
+    get isFilterDisabled () {
+        return mainLizmap.selectionTool.selectedFeaturesCount === 0 &&
+            mainLizmap.selectionTool.filteredFeaturesCount === 0;
+    }
+
+    get results () {
+        return mainLizmap.selectionTool.selectedFeaturesCount > 1 ?
```

Comment:
I would prefer having some `if else` with returns inside. It looks difficult to see there are two if nested.